src/models/MyAccount.ts

```diff
@@ -17,6 +17,13 @@ import type {
 
 let client
 
+class MyAccountModelError extends Error {
+  constructor(errorDetail, error) {
+    super()
+    this.message = `Error ${errorDetail} in ${error.filename}, line ${error.lineNumber}: ${error.message}`
```

Comment:
I think you're getting undefined for filename and lineNumber because they're not standard so it may not work on all browsers or consistently https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/lineNumber

Reply:
ah ok. i will remove.